gaurav/src/index.css

```diff
@@ -6,5 +6,116 @@
     body {
         @apply font-poppins;
     }
+  :root {
+
+        --background: 0 0% 100%;
+
+        --foreground: 0 0% 3.9%;
+
+        --card: 0 0% 100%;
+
+        --card-foreground: 0 0% 3.9%;
+
+        --popover: 0 0% 100%;
+
+        --popover-foreground: 0 0% 3.9%;
```

Comment:
remove extra lines that are empty

Reply:
Removed extra space from the `index.css` file